README.md

```diff
@@ -23,6 +23,9 @@ module.exports.bunyan = {
   /** If true, a child logger is injected on each request */
   injectRequestLogger: true,
 
+  /** If true and injectRequestLogger is enabled, then generate and attach a unique req_id to each request logger */
```

Comment:
What line length are you targetting?  I had it all on one line before the static analysis tool complained.  ;)